packages/bpk-component-calendar/src/BpkCalendarContainer.tsx

```diff
@@ -42,18 +42,18 @@ import {
 } from './date-utils';
 
 type Props = {
-  fixedWidth: boolean;
-  maxDate: Date;
-  minDate: Date;
+  fixedWidth?: boolean;
```

Comment:
have finally figured out how to handle the optional props which have default values 🙂
A bit confusing as I had read and seen examples where it says you don't need to make them optional (e.g. https://stackoverflow.com/questions/37282159/default-property-value-in-react-component-using-typescript) I think maybe this doesn't work as well with HOCs and nested components, but next best thing I found was marking them optional and using the null assertion type operator (ONLY if there is a default prop) - https://stackoverflow.com/questions/45867568/typescript-and-react-defaultprops-are-still-being-treated-as-possibly-undefined